common/rfb/ZRLEEncoder.cxx

```diff
@@ -28,15 +29,23 @@
 #include <rfb/SConnection.h>
 #include <rfb/ZRLEEncoder.h>
 #include <rfb/Configuration.h>
+#include <rfb/LogWriter.h>
 
 using namespace rfb;
 
-IntParameter zlibLevel("ZlibLevel","Zlib compression level",-1);
+static LogWriter vlog("ZRLEEncoder");
+
+IntParameter zlibLevel("ZlibLevel","[DEPRECATED] Zlib compression level",-1);
 
 ZRLEEncoder::ZRLEEncoder(SConnection* conn)
   : Encoder(conn, encodingZRLE, EncoderPlain, 127),
-  zos(0,zlibLevel), mos(129*1024)
+  zos(0, 2), mos(129*1024)
 {
+  if (zlibLevel != -1) {
+    vlog.info("Warning: The ZlibLevel option is deprecated and is "
+               "ignored by the server. The compression level can be set "
+               "by the client instead.");
```

Comment:
The indentation looks like it is off here?

Reply:
Fixed